multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -390,6 +390,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("NovelMic", "https://novelmic.com", "en", overrideVersionCode = 1),
         SingleLang("Novels Town", "https://novelstown.cyou", "ar"),
         SingleLang("Oh No Manga", "https://ohnomanga.com", "en", isNsfw = true),
+        SingleLang("Olaoe", "https://olaoe.cyou", "ar", overrideVersionCode = 1),
```

Comment:
Since it's new, you don't need to include `overrideVersionCode`

Reply:
Ok